### ISSUE TITLE
Move the StatsPlots and see about issues with distributions.

### ISSUE THREAD

**Issue by jlperla** [QuantEcon/lecture-source-jl#534]
https://github.com/JuliaPlots/StatsPlots.jl/issues/195

- [x] Try to see if this is still an issue after a package bump.
- [x] If not, then in the process of moving to `using StatsPlots` also put `using Plots` in the usual places in the documents.

**Comment by Nosferican** [comment]
If `using StatsPlots`, `using Plots` is not required.

**Comment by arnavs** [comment]
@Nosferican Are you sure? From the linked issue: 

```
Yes, there is some weird interaction going on with the reexport of Plots - always have been. You should never load Plots before StatPlots though, so that is the solution to the issue today.

The solution in the longer run will most likely be to not reexport Plots, and thus require you to load Plots.
```

**Comment by Nosferican** [comment]
https://discourse.julialang.org/t/statplots-changes-name-to-statsplots-and-plotrecipes-to-graphrecipes/20175/10?
That is how most packages work with dependencies (e.g., JuliaStats ecosystem).

**Comment by jlperla** [comment]
@arnavs  What he is saying is that using `Plots.jl` is not required.  But it is preferable in many cases for exposition, and for interactive use (i.e. you don't know if you will need statsplots until well after you have been using Plots).

We should check to see if it still an issue.  If not, then we can go back to the more common pattern we have with our includes, where we go `using Plots` at the top whereever possible, and then can locally show people the `using StatsPlots` in the cell that uses them.

**Comment by arnavs** [comment]
Roger that.

**Comment by mkborregaard** [comment]
Yes, I think `using Plots` at the beginning, then `using StatsPlots` when you need it is a completely functional pattern. As an alternative, one might consider `using StatsPlots` at the beginning and dispensing with `using Plots` altogether, given that many things I'd think quantitative economists expect to be able to do with a plotting package (plotting of data in tables, distributions, densities) actually live there.

**Comment by arnavs** [comment]
Working on this in #535

**Comment by arnavs** [comment]
Issue is resolved with #535.
